Directory.Build.props

```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
-    <DotnetVersion>net6.0</DotnetVersion>
+    <TargetDotnetVersions>net6.0;net8.0</TargetDotnetVersions>
+    <TargetDotnetVersionsWindows>net6.0-windows;net8.0-windows</TargetDotnetVersionsWindows>
```

Comment:
why this difference? you should split versions and platforms and concat then both

Reply:
trying to do that as we speak :)
having some issues, but i'll keep you posted